packages/smooth_app/lib/pages/product/edit_product_page.dart

```diff
@@ -112,7 +110,21 @@ class _EditProductPageState extends State<EditProductPage> {
                 );
                 if (refreshed ?? false) {
                   _changes++;
-                  await _refreshProduct();
+                  //Refetch product if needed for new urls
```

Comment:
Please be more generous in your comment, and explain why you do it there and not in the other places where you code `_changes++;`

Reply:
I added some more details